content/rancher-prime/azure/contents.lr

```diff
@@ -0,0 +1,282 @@
+title: Rancher Prime in Azure: Usage Instructions
+---
+theme: rancher
+---
+body:
+
+## Prerequisites <a name="prerequisites"></a>
+
+1. A Rancher-compatible AKS cluster. Please see
+   [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-6/)
+   for more details.
+2. An ingress installed on the AKS cluster so that Rancher is accessible outside
+   of the cluster. Please refer to [Rancher documentation](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-aks#5-install-an-ingress)
+   for instructions on how to deploy Ingress-INGINX on AKS cluster.
+3. The Rancher hostname must be a fully qualified domain name (FQDN) and its
+   corresponding IP address must be resolvable from a public DNS. Please refer
+   to [Rancher documentation](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-aks#7-set-up-dns)
+   for instructions on how to setup DNS.
+
+##  How to install Rancher and configure Billing-adapter.
+
+A step by step walk-through for creating a new offer from the Marketplace Overview page.
+
+1. Choose the plan from the dropdown list. ( "Plans + Pricing" tab more details about Software plan)
+2. Click 'Create' 
+
+### Basics
+
+1. Select existing subscription from the dropdown list.
+2. Select existing Resource group from the dropdown list.
+3. Select existing AKS Cluster Name from the dropdown list.
+4. Choose name for Cluster extension resource name as it suggested by the
```

Comment:
"as it suggested by the tool tip" reads odd... "as suggested" maybe? remove the "it"

Reply:
sound good